find-route-around-barriers/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.MaterialComponents.DayNight.DarkActionBar" >
```

Comment:
```suggestion
    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar" >
```
For now, samples should be set to be in light theme, we can take on supporting dark theme in a future issue.